singlestoredb/fusion/handlers/sqrl.py

```diff
@@ -0,0 +1,50 @@
+import requests
+import os
+
+from typing import Any
+from typing import Dict
+from typing import Optional
+
+from singlestoredb.fusion.handler import SQLHandler
+from singlestoredb.fusion.result import FusionSQLResult
+
+class SqrlExplain(SQLHandler):
+    """
+    SQRL EXPLAIN string_value;
```

Comment:
You should change `string_value` to something like `query` if a query string is expected here. This grammar is printed as part of the help information, so the more descriptive the labels are, the better.